projects/georeview/index.js

```diff
@@ -0,0 +1,250 @@
+      const coordsData = storageGetData(objectCoordiates);
+      const form = createForm(objectCoordiates, coordsData);
+      openBalloon(objectCoordiates, form);
+    }
+
+    // const list = storageGetData(coords);
+
+    // const form = createForm(coords, list);
+    // openBalloon(coords, form);
+  };
+
+  function createCarousel(data) {
+    // создаём болванку слайдера
+    const carouselWrap = document.createElement('div');
+    // пихаем туда вёрстку слайдера
+    carouselWrap.innerHTML = document.querySelector('#carouselTemplate').innerHTML;
+    // сюда добавляем слайды
+    const carouselContainer = carouselWrap.querySelector('.carousel');
+
+    for (const tab in data) {
+      const currentTab = data[tab];
+      // теперь каждый слайд
+      const slideWrap = document.createElement('div');
+      slideWrap.innerHTML = document.querySelector('#carouselSlideTemplate').innerHTML;
+      const slideHimself = slideWrap.querySelector('.carousel__slide');
+      const slideContent = slideWrap.querySelector('.carousel__content');
+      const button = slideWrap.querySelector('.carousel__address');
+      button.dataset.coords = currentTab.coordsId;
+      button.textContent = currentTab.address;
+
+      currentTab.data.forEach((review) => {
+        const reviewDiv = document.createElement('div');
+        reviewDiv.innerHTML = createReview(review);
+        slideContent.append(reviewDiv);
+      });
+      carouselContainer.append(slideHimself);
+    }
+
+    return carouselWrap.innerHTML;
+  }
+
+  const createPlacemark = (coords) => {
+    const placemark = new ymaps.Placemark(coords);
+
+    clusterer.add(placemark);
+  };
+
+  const createForm = (coords, reviews) => {
+    // получаем шаблон формы
+    const container = document.createElement('div');
+    container.innerHTML = document.querySelector('#addCommentForm').innerHTML;
+    const form = container.querySelector('form');
+    // находим контейнер списка отзывов
+    const reviewList = container.querySelector('.commentsList');
+    // записываем в дата атрибут координаты
+    form.dataset.coords = JSON.stringify(coords);
+
+    // перебираем отзывы и отрисовываем в балуне
+    if (reviews) {
+      for (const item of reviews) {
+        const div = document.createElement('div');
+        div.classList.add('commentsList__item');
+        div.innerHTML = createReview(item);
+        reviewList.appendChild(div);
+      }
+    }
+
+    return container.innerHTML;
+  };
+
+  const createReview = (data) => {
+    const { name, place, text, time } = data.review;
+
+    const review = `
+      <div class="commentItem">
+        <div class="commentItem__head">
+          <span class="commentItem__autor">${name}</span>
+          <span class="commentItem__info">${place}</span>
+          <span class="commentItem__info">${time}</span>
+        </div>
+        <div class="commentItem__text">${text}</div>
+      </div>
+      `;
+    return review;
```

Comment:
Можно сразу вернуть результат без создания дополнительной переменной: 
```suggestion
    return  `
      <div class="commentItem">
        <div class="commentItem__head">
          <span class="commentItem__autor">${name}</span>
          <span class="commentItem__info">${place}</span>
          <span class="commentItem__info">${time}</span>
        </div>
        <div class="commentItem__text">${text}</div>
      </div>
      `;
```